lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -42,10 +42,10 @@ def all_questions
     end
 
     def region(postcode)
-      is_london?(postcode) ? :london : :national
+      london?(postcode) ? :london : :national
     end
 
-    def is_london?(postcode)
+    def london?(postcode)
```

Comment:
Ideally this would've been included in 226d3496da925c4bdfa6f89f8911ee40a3a085fc where the method was originally added. Also the commit note could explain that removing the `is` prefix makes the method name more idiomatic Ruby.